content/posts/eks-vs-ecs.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Amazon ECS vs EKS for running CloudBees Core/Jenkins
+authors:
+  - "Logan Donley"
+date: 2020-01-13T05:05:15-04:00
+showDate: true
+tags: ["Kubernetes","aws","EKS","ECS"]
+draft: true
+---
+
+A question my colleagues and I are often asked is "if I'm already using ECS is there any point in moving to EKS (or Kubernetes in general) for running CloudBees Core?" Both Amazon ECS and EKS are great products, but we have a strong preference towards Kubernetes. We'll take a look at why that is in this post.
+
+## What are ECS and EKS?
+
+These are both services from AWS which deal with container orchestration. Containers have become a crucial component in the modern tech stack, but as your applications scale, having an orchestrator is a must. 
+
+### ECS
+
+Amazon Elastic Container Service is a platform which will orchestrate your containers in a straightforward manner. You create an application definition and it will ensure that the container is running. 
+
+With ECS there are a couple of ways you can run your containers: on EC2 instances or with [Fargate](https://aws.amazon.com/fargate/), AWS's serverless compute-engine.
+
+With EC2 you operate like you normally would and configure auto-scaling groups, networking, etc. Fargate on the other hand abstracts away the individual server and has you only worry about cpu and memory needs. The cost will be higher per compute/memory resource but it handles a lot for you.
+
+
+### EKS
+
+Amazon Elastic Kubernetes Service is AWS's managed Kubernetes platform. Kubernetes has become the community-favored container orchestrator and for good reason. It's incredibly powerful, resilient, and now has a great ecosystem built around it.
+
+With all of the power of Kubernetes comes some complexity on the infrastructure side. Setting up and maintaining your own Kubernetes cluster can be challenging even for those with lots of experience. That's why whenever possible, we would recommend you use a managed Kubernetes service like EKS, GKE, or AKS since they solve those challenges for you.
```

Comment:
Comma after challenging? "... challenging, even for those with lots of experience."